backend/isoko-backend/__tests__/unit/handlers/user/get-user-object.test.js

```diff
@@ -0,0 +1,128 @@
+const {
+    getUserObjectHandler,
+ } = require('../../../../src/handlers/user/get-user-object.js');
+ const dynamodb = require('aws-sdk/clients/dynamodb');
+ const { USER_TABLE } = require('../../../../src/constants');
+ 
+ jest.mock('aws-sdk/clients/dynamodb');
+ 
+ describe('GetUserObjectHandler tests', () => {
+    let getSpy;
+ 
+    beforeAll(() => {
+       getSpy = jest.spyOn(dynamodb.DocumentClient.prototype, 'get');
+    });
+ 
+    afterAll(() => {
+       getSpy.mockRestore();
+    });
+ 
+    afterEach(() => {
+       getSpy.mockReset();
+    });
+
+    describe('Invalid query param tests', () => {
+        it('Should throw an error when wrong HTTP method is used', async () => {
+           // arrange
+           const event = {
+              httpMethod: 'POST',
+              pathParameters: {
+                 pk: 'dd9ee02f-bc3e-45a5-b4ba-b0e2ef536573',
+              },
+           };
+  
+           // act
+  
+           // assert
+           await expect(async () => {
+              await getUserObjectHandler(event);
+           }).rejects.toThrowError();
+        });
+  
```

Comment:
once you add 400 status code responses, you'll have to update these tests to reflect that a 400 is thrown.